src/features/viewport/center-fit.ts

```diff
@@ -157,9 +159,11 @@ export abstract class BoundsAwareViewportCommand extends Command {
 
 export class CenterCommand extends BoundsAwareViewportCommand {
     static readonly KIND = CenterAction.KIND;
+    retainZoom: boolean;
```

Comment:
You don't need to add a property for `retainZoom` because the `action` is already stored as protected property in the constructor (this is indicated by the `protected` modifier of the constructor argument).

Reply:
Done, thanks